playbook-website/app/javascript/site_styles/_landing-page-solutions.scss

```diff
@@ -0,0 +1,73 @@
+.landing-page {
+  .language-card {
+    max-width: 200px;
+    flex: 1;
+
+    [class*="pb_card_kit"] {
+      height: 120px;
+      align-items: center;
+      justify-content: center;
+    }
+  }
+
+  @include break_on($screen-xl-min, 1340px) {
+    .solutionsContainer {
+      margin-right: 0 !important;
+    }
+  }
```

Comment:
This could be moved inside `.solutionsContainer` after line 42 then you don't need to repeat the selector.